docs/docsite/rst/community/index.rst

```diff
@@ -8,8 +8,72 @@ Welcome to the Ansible Community Guide!
 
 The purpose of this guide is to teach you everything you need to know about being a contributing member of the Ansible community.
 
+All types of contributions are welcome, and necessary to Ansible's continued success.
+
 To get started, please read and understand the :ref:`code_of_conduct`, and then select one of the following topics.
 
+
+Getting Started
+===============
+
+* Read the :ref:`official documentation <ansible_documentation>`
+* I want to talk to other Ansible users, how do I find a :ref:`Ansible Meetup near me <ansible_community_meetup>`.
+* I have a question, what :ref:`Ansible email lists, IRC channels <communication>` are there?
+* How can I learn more - `books <https://www.ansible.com/resources/ebooks>`_, training courses, `videos <https://www.ansible.com/resources/videos>`_ - Not sure about training courses
+* How do I get notified when new Ansible versions are released? - Subscribe to `Ansible Announce list <https://groups.google.com/forum/#!forum/ansible-announce>`_.
+
+Understanding Ansible's development process
+===========================================
+
+* I found a bug - :ref:`how to report bugs <reporting_bugs>`
+* I need functionality Ansible doesn’t offer - :ref:`requesting functionality <request_features>`_.
+* If I’m not a programmer, can I contribute?
+
+   * Help us :ref:`improve Documentation <community_documentation_contributions>`.
+   * Join a :ref:`Working Groups, and how can I get involved? <comunity_working_groups>`.
```

Comment:
There's a typo in the link - `comunity_working_groups` should be `community_working_groups`.